ManageWikiExtensions.php

```diff
@@ -2437,6 +2437,19 @@
 			'requires' => [],
 			'section' => 'other',
 		],
+		'pageassessments' => [
+			'name' => 'PageAssessments',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:PageAssessments',
+			'var' => 'wmgUsePageAssessments',
+			'conflicts' => false,
+			'requires' => [],
+			'install' => [
+				'sql' => [
+					'page_assessments' => "$IP/extensions/PageAssessments/sql/create-table--page_assessments.sql",
+					'page_assessments_projects' => "$IP/extensions/PageAssessments/sql/create-table--page_assessments_projects.sql",
+				],
```

Comment:
needs two closing arrays ], as there's two (one from install and one from sql)

Reply:
@Reception123 Yeah, thanks. I copied and pasted the wrong thing, I think.